File: python/assign-cookies.py
class Solution:
    def findContentChildren(self, g: List[int], s: List[int]) -> int:
        content_children = 0
        g.sort()
        s.sort()
        for cookie in s:
            if content_children < len(g) and g[content_children] <= cookie:
                content_children += 1
        return content_children

File: rust/assign-cookies.rs
impl Solution {
    pub fn find_content_children(mut g: Vec<i32>,mut s: Vec<i32>) -> i32 {
        let mut content_children = 0;
        g.sort();
        s.sort();
        for cookie in s {
            if content_children < g.len() && g[content_children] <= cookie {
                content_children += 1;
            }
        }
        return content_children as i32;
    }
}
